biogrid/libs/biogrid-simulator/src/lib/biobrain/biobrain-class.ts

```diff
@@ -102,17 +93,15 @@ export class BioBrain implements Brain {
   }
 
   /**
-   * This method is used for charging individual @param largeBatteries which might not have enough energy
-   * @param largeBatteries hold the large baterries which are going to be charged in case they are not full
-   * @param solarPanels holds the solar panels which are going to be used to charge @param largeBatteries
-   * @param shortestDistances holds the shortest distances between one grid item to another
+   * This method is used for charging individual largeBatteries which might not have enough energy
    * @return calls the @method determineSupplyingPath which returns @interface SupplyingPath
```

Comment:
This line should describe the return value, not the function as a whole

Reply:
Thank you. I will adjust it now